packages/rocketchat-ui-message/client/messagePreview.html

```diff
@@ -0,0 +1,31 @@
+<template name="messagePreview">
```

Comment:
Why do you need this component? Did you try to use the message render like we do here https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-integrations/client/views/integrationsIncoming.html#L119